server/pulp/server/managers/consumer/applicability.py

```diff
@@ -118,29 +120,52 @@ def regenerate_applicability_for_repos(repo_criteria):
         repo_criteria.fields = ['id']
         repo_ids = [r.repo_id for r in model.Repository.objects.find_by_criteria(repo_criteria)]
 
+        task_group_id = uuid4()
+
         for repo_id in repo_ids:
             # Find all existing applicabilities for given repo_id. Setting batch size of 5 ensures
```

Comment:
This comment can be removed. Since this function no longer does the work synchronously, it should never cause that timeout.